spec/helpers/quadicon_helper_spec.rb

```diff
@@ -98,6 +98,8 @@
 # FIXME: complex describe blocks mirror the existing complex control flow
 
 describe QuadiconHelper do
+  helper(ImageEncodeHelper)
```

Comment:
Super-minor, but `helper` here is a DSL-like method, which usually we avoid the parens on